public/app/plugins/datasource/grafana-azure-monitor-datasource/grafanaTemplateVariableFns.ts

```diff
@@ -258,7 +258,7 @@ const migrateGrafanaTemplateVariableFn = (query: AzureMonitorQuery) => {
     return query;
   }
 
-  const migratedQuery: AzureMonitorQuery = {
```

Comment:
this can still be a const?

Reply:
Indeed, I thought I had to change it when I had the `delete` for `grafanaTemplateVariableFn` but you're right 👍